hack/vendor.sh

```diff
@@ -69,7 +69,13 @@ mv tmp-digest src/github.com/docker/distribution/digest
 mkdir -p src/github.com/docker/distribution/registry
 mv tmp-api src/github.com/docker/distribution/registry/api
 
+# This test file checks that resumable support is available in digest library.
+# This is not at all needed in the docker daemon at this time. We remove it to
+# avoid compile-time issues with tests.
+rm -f src/github.com/docker/distribution/digest/digester_resumable_test.go # dependencies not required.
```

Comment:
no but thats kinda gross tbh, we dont do that for any other deps what are
packagers supposed to do

On Fri, May 29, 2015 at 3:02 PM, Stephen Day <notifications@github.com>
wrote:

> In hack/vendor.sh
> <https://github.com/docker/docker/pull/13576#discussion_r31368677>:
>
> > @@ -69,7 +69,13 @@ mv tmp-digest src/github.com/docker/distribution/digest
> >  mkdir -p src/github.com/docker/distribution/registry
> >  mv tmp-api src/github.com/docker/distribution/registry/api
> >
> > +# This test file checks that resumable support is available in digest library.
> > +# This is not at all needed in the docker daemon at this time. We remove it to
> > +# avoid compile-time issues with tests.
> > +rm -f src/github.com/docker/distribution/digest/digester_resumable_test.go # dependencies not required.
>
> Do we run tests of vendored code?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/13576/files#r31368677>.
>